<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 255);  /* 흰색 */
    border-radius: 5px;
}

QMenuBar {
    background-color: rgba(245, 245, 245, 255);  /* 밝은 회백 */
    color: #333;
    font-weight: bold;
    border: none;
}

QMenuBar::item {
    background: transparent;
    padding: 5px 15px;
}

QMenuBar::item:selected {
    background: rgba(220, 220, 220, 255);  /* 연한 회색 */
    color: black;
}

QMenu {
    background-color: rgba(250, 250, 250, 255);
    color: #333;
}

QMenu::item:selected {
    background-color: rgba(230, 230, 230, 255);
    color: black;
}

QPlainTextEdit {
    background-color: rgba(255, 255, 255, 255);
    border: 1px solid rgba(200, 200, 200, 255);
    border-radius: 6px;
    padding: 10px;
    color: #222;
    font-family: &quot;맑은 고딕&quot;, sans-serif;
    font-size: 13px;
}

QToolBar {
    background-color: {{MENU_BG}};  /* 메뉴바와 동일하게 */
    spacing: 4px;
    padding: 4px;
    margin: 0px;
    border: none;
}

QStatusBar {
    background-color: transparent;
    margin: 0px;
    padding: 2px;
    color: #444;
}

QMessageBox {
    background-color: white;
    color: black;
    border-radius: 6px;
}

QMessageBox QLabel {
    background-color: transparent;
    color: black;
}

/* ✅ 일반 버튼 */
QPushButton {
    background-color: rgba(240, 240, 240, 255);
    border: 1px solid rgba(200, 200, 200, 255);
    border-radius: 6px;
    padding: 6px 12px;
    color: black;
}

QPushButton:hover {
    background-color: rgba(220, 220, 220, 255);
    color: black;
	border-radius: 5px;
}

/* ✅ 타이틀바 버튼 (min/max/close) - 메뉴바 버튼 스타일로 */
QPushButton#TitleButton {
    background-color: transparent;
    border: none;
    padding: 4px 8px;
    color: #222;
    font-size: 12px;
    font-weight: normal;
    border-radius: 0px;
}

QPushButton#TitleButton:hover {
    background-color: rgba(220, 220, 220, 255) ;  /* 메뉴바 hover와 동일 색상 */
    color: white;
}


QMessageBox QPushButton {
    background-color: rgba(230, 230, 230, 255);
    border: 1px solid rgba(200, 200, 200, 255);
    border-radius: 5px;
    padding: 5px 10px;
    color: #222;
    font-weight: bold;
}

QMessageBox QPushButton:hover {
    background-color: rgba(210, 210, 210, 255);
    color: black;
}
QToolButton {
    background-color: transparent;
    border: none;
    padding: 2px;
    margin: 0px;
}

QToolButton:hover {
    background-color: rgba(150, 160, 255, 40);  /* 살짝 하이라이트 효과 */
    border-radius: 16px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="plain_te"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuForm">
    <property name="title">
     <string>Form</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionTheme"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWidgets">
    <property name="title">
     <string>Widgets</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuForm"/>
   <addaction name="menuHelp"/>
   <addaction name="menuWidgets"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionTheme"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>img/new.png</normaloff>img/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>img/open.png</normaloff>img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>img/save.png</normaloff>img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normaloff>img/save_as.png</normaloff>img/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>img/exit.png</normaloff>img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>img/copy.png</normaloff>img/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>img/paste.png</normaloff>img/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>img/delete.png</normaloff>img/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>img/cut.png</normaloff>img/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset>
     <normaloff>img/font.png</normaloff>img/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>img/about.png</normaloff>img/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>img/undo.png</normaloff>img/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset>
     <normaloff>img/select_all.ico</normaloff>img/select_all.ico</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>img/redo.png</normaloff>img/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="icon">
    <iconset>
     <normaloff>img/theme.png</normaloff>img/theme.png</iconset>
   </property>
   <property name="text">
    <string>Theme</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionMugic_Player">
   <property name="text">
    <string>Mugic Player</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
